plugins/commands/reporter/src/main/kotlin/ReporterCommand.kt

```diff
@@ -295,7 +295,7 @@ class ReporterCommand : OrtCommand(
                         reporter to measureTimedValue {
                             val options = reportConfigMap[reporter.descriptor.id] ?: PluginConfiguration.EMPTY
                             reporter.create(PluginConfig(options.options, options.secrets))
-                                .generateReport(input, outputDir, options)
+                                .generateReport(input, outputDir)
```

Comment:
This commit should also drop the various now unneeded `this.` qualifiers.

Reply:
Good point, I expected Detekt would complain about it. I hope I found all occurrences.